.travis.yml

```diff
@@ -34,7 +34,7 @@ script:
     fi
   - cd fixed-hash/ && cargo test --all-features && cd ..
   - cd uint/ && cargo test --features=std,quickcheck --release && cd ..
-  - cd plain_hasher/ && cargo test --no-default-features && cd ..
+  - cd plain_hasher/ && cargo test --no-default-features && cargo test --benches && cd ..
```

Comment:
```suggestion
  - cd plain_hasher/ && cargo test --no-default-features && cargo check --benches && cd ..
```

Reply:
It just my personal convention. `cargo test --benches` is building all benchmarkable targets and running them as tests (include running the unit-tests). Maybe we can do `cargo test --benches -- --bench` which will ignore running unit-tests.

Reply:
Actually, it runs the benchmarks like that (include running unit-tests):
```
running 1 test
test tests::it_works ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out

...

Testing write_plain_hasher
Success

Testing write_default_hasher
Success
```

Reply:
Right, and we do not wish to run the benchmarks on CI.

(Not that it's a bad thing per se to run the benches, but it's not worth anything if we don't collect the output performance data and store it somewhere, and we're not so for the time being running the benches just slows CI down for no benefit.)